projects/legacy/components/multi-select/multi-select-group/multi-select-group.directive.ts

```diff
@@ -28,16 +29,24 @@ export const TUI_MULTI_SELECT_OPTION = new PolymorpheusComponent(
                 const multiSelect = inject(TuiMultiSelectComponent, {optional: true});
                 const ngControl = inject(NgControl);
                 const host = inject(AbstractTuiControl, {optional: true});
+                const {identityMatcher, stringify} =
+                    inject(TUI_ITEMS_HANDLERS, {optional: true}) ?? {};
 
                 return (
                     multiSelect || {
+                        stringify,
+                        identityMatcher,
                         handleOption: (option) => {
+                            const state = tuiArrayToggle(
+                                host ? host.value : ngControl.control?.value || [],
```

Comment:
Looks like we should make a const for this line:
```ts
const value = host?.value ?? ngControl.control?.value ?? [];
```

Reply:
fixed